compose.yaml

```diff
@@ -76,7 +76,8 @@ services:
       - ./collaboration-service:/usr/src/collaboration-service
       - /usr/src/collaboration-service/node_modules
     ports:
-      - "${COLLABORATIONSVC_PORT:-3001}:3001"
+      - "${COLLABORATIONSVC_PORT:-3001}:3001" # WebSocket server
+      - "3002:3002"  # REST API server
```

Comment:
change this line to maintain consistency with the other docker compose ports: 
`"${COLLABORATIONSVCRESTAPI_PORT:-3002}:3002" # REST API server`

Reply:
oki done